satellite/payments/stripecoinpayments/transactions_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package stripecoinpayments_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/payments/coinpayments"
+	"storj.io/storj/satellite/payments/stripecoinpayments"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func TestTransactionsDB(t *testing.T) {
+	satellitedbtest.Run(t, func(t *testing.T, db satellite.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		transactions := db.CoinpaymentsTransactions()
+
+		t.Run("test insert", func(t *testing.T) {
```

Comment:
No need to rewrite `"test"`, since it's running in a test, it's obviously a test :)

Reply:
makes sense)